ios/ExposureProcessor.swift

```diff
@@ -123,19 +128,41 @@ public class ExposureProcessor {
         }
         self.scheduleCheckExposure()
     }
+
+    public func pause(_ resolve: @escaping RCTPromiseResolveBlock,
+                        rejecter reject: @escaping RCTPromiseRejectBlock) {
+        guard ENManager.authorizationStatus == .authorized else {
+            os_log("Not authorised so can't pause", log: OSLog.exposure, type: .info)
+            return reject("NOTAUTH", "Not authorised to start", nil)
+        }
+        let context = Storage.PersistentContainer.shared.newBackgroundContext()
+        Storage.shared.flagPauseStatus(context, true)
```

Comment:
This should be included at L145 so as to not be stored as true - in the event the `setExposureNotificationEnabled` throws

Reply:
Needs to be set before the call or otherwise the state changes that happen as a result of stopping won'tbe flagged as pause events.
Will add a line if it fails to remove the pause flag.